ui/src/components/credentials/IssueCredential.tsx

```diff
@@ -41,6 +41,7 @@ const defaultCredentialFormInput: CredentialFormInput = {
   },
   issueCredential: {
     proofTypes: ["SIG"],
+    refreshService: { enabled: false, url: "" },
```

Comment:
`url` should not be required when enabled is false.

Reply:
Please, see previous comment https://github.com/0xPolygonID/issuer-node/pull/604#discussion_r1464608691 I do not know how to make url nullable. But I don't see a risk in having a default value of empty string.